### DISCUSSION TITLE
Using PSRAM on ESP32 WROVER B

### DISCUSSION THREAD

**Discussion by DiegoEstradaS** [Bodmer/TFT_eSPI#1413] [Q&A - Sprites]
How can i implement the use on PSRAM for creating Sprites and maybe loading fonts?
I've seen the following lines in the master file but i dont know how to implement it:
_psram_enable;
 #define PSRAM_ENABLE 3
Is there any example on the library that uses PSRAM? I'm using the TTGO T-CALL board with ESP32 WROVER B.

**Comment by Bodmer** [comment]
If the ESP32 board has PSRAM and it is enabled in the Arduino IDE then sprites automatically use PSRAM unless you disable it. 

When a smooth font is loaded then the loaded font data will also automatically use PSRAM unless you disable it.
